src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -423,31 +423,47 @@ private static async Task<Component[]> GetComponents(string[] urls)
 
                 var fileName = urlString.Split(';').Last();
                 var name = fileName.Remove(fileName.Length - 6, 6);
-                var version = await GetVersionFromComponentUrl(uri);
+                var localFilePath = buildArtifacts.FindFilePath(fileName);
```

Comment:
do you have an example of finding the path?  I'm not 100% certain why we can't 'build' the path if that makes sense

Reply:
The vsman files do not seem to be in a consistent location other than in the VSSetup artifact. Sometimes you will find them in the DevDivPackages subfolder, sometimes in an Insertions subfolder.